common/scala/src/main/scala/whisk/core/entity/EntityPath.scala

```diff
@@ -45,9 +45,30 @@ protected[core] class EntityPath private (val path: Seq[String]) extends AnyVal
     def addpath(e: EntityName) = EntityPath(path :+ e.name)
     def root = EntityPath(Seq(path(0)))
     def last = EntityName(path.last)
+    def dropLast = EntityPath(path.dropRight(1)) // remove last element from path (e.g., name)
+    def defaultPackage = path.size == 1   // if only one element in the path, then it's the namespace with a default package
```

Comment:
I believe this is dangerous or misleading: We currently abuse this type everywhere to express FQN's instead of more typesafe ones. `myPackage/test` will match this as well while not being the default package.